### DISCUSSION TITLE
Prefixes aggregation and filtering

### DISCUSSION THREAD

**Discussion by hamshiva** [akvorado/akvorado#217] [Q&A]
Hi,

I wonder if adding the possibility to use src and dst network prefixes as dimension and filter would be possible ?
I know that netflow gives srcMask and dstMask fields that could be used for that, but right now it seems that the inlet is not using those fields.

From my point of view this could be very useful for routing optimizations/decisions as obviously routing is done using network prefixes and not single addresses (/32 or /128).

What is your opinion about this ?

**Comment by vincentbernat** [comment]
We can add SrcNetwork/DstNetwork. Please, open an enhancement request. The only downside in adding more fields is that the protobuf serialization can become a bit huge. ClickHouse shouldn't care much when the fields are empty and they don't alter performance if they are not used.
